tests/e2e/device-plugin_test.go

```diff
@@ -168,25 +148,23 @@ var _ = NVDescribe("GPU Device Plugin", func() {
 				Expect(err).NotTo(HaveOccurred())
 
 				targetNodeName := nodes[0].Name
```

Comment:
Out of scope for this PR: Looking at this, is `nodes` always guaranteed to have at least one element?

Reply:
`GetNonControlPlaneNodes` ensures it doesn't or returns err